### PR TITLE
Snowboy library integration into our system

### PR DESCRIPTION
**Pull request by BillWSY**

### CODE REVIEW

File: codelab/snowboy/demo.cc [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 // Copyright 2016  KITT.AI (author: Guoguo Chen)
 
-// Make sure this is the first included file
-
+// Make sure this line is included (and is the first line) of any cc_library
+// uses snowboy, see https://github.com/Kitt-AI/snowboy/issues/81
```
Comment (zephirefaith) [inline codelab/snowboy/demo.cc]: Let's add this fact to the wiki too.

Comment (BillWSY) [inline codelab/snowboy/demo.cc]: Yes. In addition, we will make a wrapper library for their library. The interface between them must only be POD type. So user and maintainers would not touch this nasty part.

Comment (zephirefaith) [inline codelab/snowboy/demo.cc]: That sounds good! Can I do that? I want to get some hands on experience with the non ROS part of the code..

**Comment by BillWSY** [comment]
Just tested, both keywords (Alexa and Snowboy) works.

**Pull request merged**